app/qml/RegistrationForm.qml

```diff
@@ -212,7 +212,7 @@ Rectangle {
         height: fieldHeight
         width: registerForm.width
         text: qsTr("I accept the Mergin %1Terms and Conditions%3 and %2Privacy Policy%3")
-              .arg("<a href='"+__merginApi.apiRoot + "assets/mergin_terms_of_service.pdf'>")
+              .arg("<a href='"+__merginApi.apiRoot + "assets/tos.html'>")
```

Comment:
There is already `__inputHelp.merginTermsLink`, let's use that one instead

Reply:
Good point! However, `__inputHelp.merginTermsLink` uses a hardcoded Mergin api root pointing at public Mergin server. If we want to support CE Mergin server, its probably better to use current client's mergin api root (`__merginApi.apiRoot`).